Adding docs about generating metadata feature in integration studio (#4790)
wso2/docs-apim: en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md

## Let's get started!

Follow the steps given below to build this use case and try it out.
    !!! Note

>         See the [Generating Service Catalog Metadata Artifact]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md) for more information to generate metadata artifacts if you are using an older Integration Studio workspace.

Reviewer: ```suggestion
        For more information on generating metadata artifacts if you are using an older Integration Studio workspace, see [Generating Service Catalog Metadata Artifact]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md).
```

Author: Fixed
